llvm_objects.py

```diff
@@ -162,7 +205,7 @@ def _get_instructions(self, l_block, w_module):
             prev = current_instr
             w_instrs.append(current_instr)
             l_instr = llwrap.LLVMGetNextInstruction(l_instr)
-        return w_instrs
+        return w_instrs[:]
```

Comment:
Does everyone who call this function expect to mutate the list? If not, should the caller be responsible for cloning the list?